components/AllocatedWorkers/DeallocateWorkerDetails/DeallocatedWorkerDetails.jsx

```diff
@@ -21,14 +21,17 @@ const DeallocatedWorkersDetails = ({
   const [complete, setComplete] = useState(false);
   const [error, setError] = useState(false);
   const [deallocationReason, setDeallocationReason] = useState('');
+  const [worker] = useState(allocatedWorker);
+  const [type] = useState(workerType);
+  const [workerTeam] = useState(allocatedWorkerTeam);
```

Comment:
we don't need to store these anymore as we are closing the modal.

Reply:
Should we not keep them for the future when we will use the confirmation screen?